assets/js/frontend-form.js

```diff
@@ -106,6 +107,14 @@
             });
         },
 
+        handleReadOnly: function(){
+            $('.wpuf-form > .read-only').each( function () {
+                $('input, select, textarea', this).each( function () {
+                     $(this).attr( 'disabled', true );
```

Comment:
I haven't run the code, but we are disabling the fields here. Is there any code to enable the disabled ones again? If it works, no problem.

Reply:
There isn't any code to enable this. as the requirement it is not like a toggle. the field will be disabled if the admin wants to.